plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -50,6 +48,16 @@ struct permission {
 template<typename>
 struct resolver_factory;
 
+template<typename Type>
+Type convert_to_type(const string& str, const string& desc) {
+   try {
+      return fc::variant(str).as<Type>();
+   } FC_RETHROW_EXCEPTIONS(warn, "Could not convert {desc} string '${str}' to key type.", ("desc", desc)("str",str) )
```

Comment:
`{desc}` should be `${desc}`. Other than that, it looks good.

Reply:
ugh, that is what I get for hurrying.